db/schema.rb

```diff
@@ -42,6 +42,11 @@
     t.index ["blob_id", "variation_digest"], name: "index_active_storage_variant_records_uniqueness", unique: true
   end
 
+  create_table "java_scripts", force: :cascade do |t|
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
```

Comment:
What is the purpose of this table?

Reply:
I already corrected it. Should not exist anymore :)